lib/resources/r4/observation.yaml

```diff
@@ -414,13 +414,45 @@ fields:
                 "text": "primary performer"
              },
              "url": "http://hl7.org/fhir/StructureDefinition/event-performerFunction"
+          },
+          {
+             "valueCodeableConcept":{
+                "coding":[
+                   {
+                      "system":"http://terminology.hl7.org/CodeSystem/v3-ParticipationType",
+                      "code":"PRF",
+                      "display":"performer"
+                   }
+                ],
+                "text":"performer"
+             },
+             "url":"http://hl7.org/fhir/StructureDefinition/event-performerFunction"
+          },
+          {
+             "valueCodeableConcept":{
+                "coding":[
+                   {
+                      "system":"http://terminology.hl7.org/CodeSystem/v3-ParticipationType",
+                      "code":"AUTHEN",
+                      "display":"authenticator"
+                   }
+                ],
+                "text":"authenticator"
+             },
+             "url":"http://hl7.org/fhir/StructureDefinition/event-performerFunction"
```

Comment:
Is this the labs block?

Reply:
We aren't differentiating the blocks here. The goal is to show how each performer looks like